virt/install/preparing-cluster-for-virt.adoc

```diff
@@ -53,27 +64,39 @@ include::snippets/technology-preview.adoc[leveloffset=+2]
 
-* When using a disconnected cluster on a restricted network, you must xref:../../operators/admin/olm-configuring-proxy-support.adoc#olm-configuring-proxy-support[configure proxy support in Operator Lifecycle Manager] to access the Red Hat-provided OperatorHub. Using a proxy allows the cluster to fetch the {VirtProductName} Operator.
+== Computing requirements
 
-* If your cluster uses worker nodes with different CPUs, live migration failures can occur because different CPUs have different capacities. To avoid such failures, use CPUs with appropriate capacity for each node and set node affinity on your virtual machines to ensure successful migration. See xref:../../nodes/scheduling/nodes-scheduler-node-affinity.adoc#nodes-scheduler-node-affinity-configuring-required_nodes-scheduler-node-affinity[Configuring a required node affinity rule] for more information.
+All CPUs must be supported by Red Hat Enterprise Linux 8 and meet the following requirements:
 
-* All CPUs must be supported by Red Hat Enterprise Linux 8 and meet the following requirements:
+* Intel 64 or AMD64 CPU extensions are supported
+* Intel VT or AMD-V hardware virtualization extensions are enabled
+* The no-execute (NX) flag is enabled
 
-** Intel 64 or AMD64 CPU extensions are supported
-** Intel VT or AMD-V hardware virtualization extensions are enabled
-** The no-execute (NX) flag is enabled
+If your cluster uses worker nodes with different CPUs, Live Migration failures can occur because different CPUs have different capacities. To avoid these failures, use CPUs with appropriate capacity for each node and set node affinity on your virtual machines to ensure successful migration. See xref:../../nodes/scheduling/nodes-scheduler-node-affinity.adoc#nodes-scheduler-node-affinity-configuring-required_nodes-scheduler-node-affinity[Configuring a required node affinity rule] for more information.
 
-* If FIPS mode is xref:../../installing/installing-fips.adoc#installing-fips[enabled for your cluster], no additional setup is needed for {VirtProductName}. Support for FIPS cryptography must be enabled before the operating system that your cluster uses boots for the first time.
+== Scaling requirements
 
-{VirtProductName} works with {product-title} by default, but the following installation configurations are recommended:
+You must manage Compute nodes according to the number and size of the virtual machines that you want to host in the cluster.
 
-* Configure xref:../../monitoring/monitoring-overview.adoc#monitoring-overview[monitoring] in the cluster.
+See xref:../../virt/install/virt-planning-environment-object-maximums.adoc#virt-planning-environment-object-maximums[Planning your environment according to {VirtProductName} object maximums] for more information.
+
+== Requirements for deploying in {VirtProductName} disconnected environments
```

Comment:
s/in {VirtProductName}/{VirtProductName} in

Reply:
Done.